content/soda-gettingstarted/multicloud-ha-installation-using-ansible.md

```diff
@@ -0,0 +1,71 @@
+---
+title: SODA Multi Cloud HA - Installation Guide using Ansible
+description: "This document describes how to install SODA Multi Cloud project on a pre-configured Kubernetes cluster. These steps will help you to install / uninstall Multi-Cloud project. After installation using these steps, SODA Multi-Cloud Services will be available in HA mode. You can test either through SODA Dashboard UI or CLI"
+weight: 50
+---
+
+This document describes how to install SODA Multi Cloud project on a pre-configured Kubernetes cluster. These steps will help you to install / uninstall Multi-Cloud project. After installation using these steps, SODA Multi-Cloud Services will be available in HA mode. You can test either through SODA Dashboard UI or CLI
+
+## Pre-config for Deploying Multi-Cloud Services on Kubernetes Cluster (Ubuntu 16.04 or Ubuntu 18.04)
+All the installation work is tested on `Ubuntu 16.04` and `Ubuntu 18.04`, please make sure you have installed the right one. Also `root` user is REQUIRED before the installation work starts.
+The following configuations are required on the Kubernetes Cluster and the SODA installer node. 
+  1. A Kubernetes Cluster should be available for deploying the SODA Multi-Cloud Services on it.
+  2. The Kubernetes Cluster should have atleast three nodes to deploy the services. 
+  3. The Kubernetes Master node should have a user, (say 'k8s_user') configured to access the installer node.
+  4. [Passwordless SSH](https://help.ubuntu.com/community/SSH/OpenSSH/Keys) should be setup between the Kubernetes Master node and the SODA Installer node.
+  [TODO: Add minimum configuration for K8s nodes.]
+
+
+All the other configurations remain same as the [standalone installation](installation-using-ansible.md). 
+Please follow the standalone installation guide for the remaining installation steps. 
+Here are the steps that are specific to the HA installation of Multi-Cloud Services on Kubernetes Cluster.
+
+
+### Configurations for Installing Multi-Cloud in HA mode.     
+The basic configurations for installing SODA Multi-Cloud in HA mode are the same as installing Multi-Cloud in the standalone mode. The few additional steps required to install Multi-Cloud in HA mode are 
+
+As a root user
+#### Download SODA installer code
+```bash
+git clone https://github.com/sodafoundation/installer.git
+cd installer/ansible
+# Checkout the required version. For example, to checkout v1.2.0 follow
+git checkout v1.2.0
+```
+{{% notice warning %}}
```

Comment:
@rajat-soda Since this is only going to be HA installation guide for Multicloud only and it is a configuration change in the regular installation process I think it can be part of the [recommended installation guide](https://docs.sodafoundation.io/soda-gettingstarted/installation-using-ansible/) using ansible. 

I see most of the steps are repeated and it will be easier for the user to just follow the guide.
the steps for enabling HA and the associated K8S changes can be a section that can be linked externally too by using a heading. 

@kumarashit @skdwriting Let me know your thoughts.

Reply:
IMO, this can be a part of the recommended installation guide, there are steps to configure different projects in the installation guide, same way to configure HA for multicloud there can be a link which re-directs to this page

Reply:
Done.